binary_to_decimal/binary_to_decimal.js

```diff
@@ -0,0 +1,53 @@
+// ----------------------------------
+// Function binaryToDecimal
+
+function binaryToDecimal(binary_number){
+    const length = binary_number.toString().length; 
+    const string = binary_number.toString();
```

Comment:
Převody na string jsou zde zbytečné. binary_number proměnná už přichází jako string z promptu.

Reply:
Vďaka, to by ma nenapadlo, som zvyknutý na C a deklarácie premenných, pričom v JavaScripte sa variables dynamicky priradzujú.

Reply:
Proto se samotný JS už dnes na nových projektech téměř nepoužívá a dává se přednost TypeScriptu. Je to slabina JS